x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/advanced_step_form.tsx

```diff
@@ -481,18 +485,35 @@ export const AdvancedStepForm: FC<CreateAnalyticsStepProps> = ({
               }
             )}
           >
-            <EuiFieldText
-              placeholder={
-                jobType !== undefined
-                  ? DEFAULT_MODEL_MEMORY_LIMIT[jobType]
-                  : DEFAULT_MODEL_MEMORY_LIMIT.outlier_detection
-              }
-              disabled={isJobCreated}
-              value={modelMemoryLimit || ''}
-              onChange={(e) => setFormState({ modelMemoryLimit: e.target.value })}
-              isInvalid={modelMemoryLimitValidationResult !== null}
-              data-test-subj="mlAnalyticsCreateJobWizardModelMemoryInput"
-            />
+            <>
+              <EuiFieldText
+                placeholder={
+                  jobType !== undefined
+                    ? DEFAULT_MODEL_MEMORY_LIMIT[jobType]
+                    : DEFAULT_MODEL_MEMORY_LIMIT.outlier_detection
+                }
+                disabled={isJobCreated || useEstimatedMml}
+                value={useEstimatedMml ? estimatedModelMemoryLimit : modelMemoryLimit || ''}
+                onChange={(e) => setFormState({ modelMemoryLimit: e.target.value })}
+                isInvalid={modelMemoryLimitValidationResult !== null}
+                data-test-subj="mlAnalyticsCreateJobWizardModelMemoryInput"
+              />
+              <EuiSpacer size="s" />
+              <EuiSwitch
+                disabled={isJobCreated}
+                name="mlDataFrameAnalyticsUseEstimatedMml"
+                label={i18n.translate('xpack.ml.dataframe.analytics.create.UseEstimatedMmlLabel', {
```

Comment:
super duper nit: `create.useEstimatedMmlLabel` here to match the convention

Reply:
Updated in 66bcffa8bbd0ae7d3eec6847b4085fc2c4e86d35 
👍  😄